examples/speaker_tasks/recognition/extract_speaker_embeddings.py

```diff
@@ -53,29 +51,22 @@ def autocast(enabled=None):
 
 
 def get_embeddings(speaker_model, manifest_file, batch_size=1, embedding_dir='./', device='cuda'):
-    test_config = OmegaConf.create(
-        dict(manifest_filepath=manifest_file, sample_rate=16000, labels=None, batch_size=batch_size, shuffle=False,)
-    )
-
-    speaker_model.setup_test_data(test_config)
-    speaker_model = speaker_model.to(device)
-    speaker_model.eval()
-
-    all_embs = []
-    out_embeddings = {}
-
-    for test_batch in tqdm(speaker_model.test_dataloader()):
-        test_batch = [x.to(device) for x in test_batch]
-        audio_signal, audio_signal_len, labels, slices = test_batch
-        with autocast():
-            _, embs = speaker_model.forward(input_signal=audio_signal, input_signal_length=audio_signal_len)
-            emb_shape = embs.shape[-1]
-            embs = embs.view(-1, emb_shape)
-            all_embs.extend(embs.cpu().detach().numpy())
-        del test_batch
-
+    """
```

Comment:
Good catch, previously this function was used to have forward loop, hence autocast was employed to get fp16 embeddings. Now since I am re-using the function call from class, this no is required.

Reply:
Got it.